frontend/src/components/TicketList.js

```diff
@@ -45,7 +53,9 @@ const TicketList = (props) => {
                             borderRadius="0"
                             mt="0 !important"
                             onClick={() =>
-                                navigate(`/${ticket.type}/${ticket.id}`)
+                                navigate(
+                                    `/${ticket.type}/${ticket.id}?tickettypes=${tickettypes}`
```

Comment:
ensure that all instances of `useNavigate` and `navigate` preserve these query params (mainly ticket tree view)